pyuvsim/tests/test_mpi.py

```diff
@@ -188,18 +191,21 @@ def test_big_bcast_gather_loop(fake_tasks):
 
 
 @pytest.mark.parallel(3)
-def test_sharedmem_bcast_with_quantities():
-    # Use mpi.quantity_shared_bcast and check returned objects.
+def test_sharedmem_bcast():
```

Comment:
The point of this test is to verify that the quantity_shared_bcast function works properly with astropy Quantities. Do we no longer need that feature?

Reply:
The only place that the `quantity_shared_bcast` function was called was in this test. It's not used in the code, I assume because of the conversion from using a SkyModel object to using a SkyModelData object? So we removed it and converted this test to testing the underlying `sharedmem_bcast` function.